Classes/OldModel/Event.php

```diff
@@ -389,27 +389,45 @@ public function getPlacesWithCountry(): array
             return [];
         }
 
-        $countries = [];
-
-        $countryData = \Tx_Oelib_Db::selectMultiple(
-            'country',
-            'tx_seminars_sites LEFT JOIN ' .
-            'tx_seminars_seminars_place_mm ON tx_seminars_sites' .
-            '.uid = tx_seminars_seminars_place_mm.uid_foreign' .
-            ' LEFT JOIN tx_seminars_seminars ON ' .
-            'tx_seminars_seminars_place_mm.uid_local = ' .
-            'tx_seminars_seminars.uid',
-            'tx_seminars_seminars.uid = ' . $this->getUid() .
-            ' AND tx_seminars_sites.country <> ""' .
-            \Tx_Oelib_Db::enableFields('tx_seminars_sites'),
-            'country'
-        );
-
-        foreach ($countryData as $country) {
-            $countries[] = $country['country'];
-        }
+        $queryBuilder = self::getQueryBuilderForTable('tx_seminars_sites');
+        $countryData = $queryBuilder
+            ->select('country')
+            ->from('tx_seminars_sites')
+            ->from('tx_seminars_seminars_place_mm')
+            ->from('tx_seminars_seminars')
```

Comment:
These multiple `from` calls do not look right.

Reply:
All three tables are used in this query so they need to be inside the `FROM` part of the SQL query. As the `from()` method does not allow multiple params, calling `from()` multiple times is the only way to archive this.